cordapp-cpk2/src/test/kotlin/net/corda/plugins/cpk2/json/CPKDependency.kt

```diff
@@ -0,0 +1,14 @@
+package net.corda.plugins.cpk2.json
+
+class CPKDependencyFile(
+    val formatVersion: String,
+    val dependencies: Array<CPKDependency>
```

Comment:
`Array<CPKDependency>` is an odd choice here when you would clearly prefer to use `List<CPKDependency>` :thinking:. Why not use `List<CPKDependency>` here directly instead?

Reply:
refactored